notification/src/handler/notification/notification.handler.js

```diff
@@ -312,9 +342,11 @@ function getAddTransactionUpdateAction({
   currency,
   interactionId,
 }) {
+  const currentTimestamp = new Date().toISOString()
```

Comment:
Why we generate instead of taking it from Adyen notification?

Reply:
Get `eventDate` from notification and apply it in `addTransaction` update action
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/8dc3daa1e3fd8001d933c30e260cdd3794e675be